census/ui_app/HelperFields.jsx

```diff
@@ -1,6 +1,22 @@
 import _ from 'lodash';
 import React from 'react';
 
+const DiscussionLink = props => {
+  if (props.isReview && props.url) {
+    return (
+    <div className="submit continuation question">
+      <div className="main">
+        <div>
+          <div className="instructions"></div>
+          <p>Do you want to discuss this submission? Start a <a target="_blank" href={props.url}>new topic</a> on the forum!</p>
```

Comment:
I have a small issue with the wording here. Not that is doesn't accurately describe what is does, but it's not really inviting. I wouldn't imagine many people would click on it to discuss submissions on the forum, and that's a pity, given the amazing platform Discourse is :)

But I don't have any suggestions, so feel free to leave it like this if nothing crosses your mind.